org.openhab.binding.zigbee/README.md

```diff
@@ -183,6 +183,16 @@ If the USB dongle is not recognized, it might be necessary to make the dongle's
   ```
 - Plug in the dongle. It should now be recognized properly as ttyUSBx.
 
+##### EmberZNet and EZSP Protocol Version
+
+Silicon Labs (a.k.a. Silabs) Zigbee stack is called "EmberZNet PRO" (formerly "Ember Zigbee") and provides "EmberZNet Serial Protocol" (EZSP) as the protocol for a host application to interact with. openHAB is currenty support EZSP v5, v6, v7, and v8 which covers most if not all available Zigbee dongle based on Silabs Zigbee chips.
+
+Silicon Labs does not currently have a consolidated list of changes by EmberZNet SDK or EZSP protocol version. The EZSP additions, changes and deletions have only ever been listed in the "Zigbee EmberZNet Release Notes" (EmberZNet SDK) under the "New items" section as well as the Silabs UG100 EZSP Reference Guide included with each EmberZNet SDK release.
+
+Perhaps more important to know is that EZSP v5, v6 and v7 (EmberZNet 6.6.x.x) use the same framing format, while EZSP v8 (EmberZNet 6.7.x.x) introduced a new framing format and expanded command id field from 8 bits to 16 bits and is therefor not backward compatible. 
+
+The largest change was between EZSP v4 (first added in EmberZNet 4.7.2 SDK) and EZSP v5 that was added in EmberZNet 5.9.0 SDK which requires the Legacy Frame ID 0xFF. The change from EZSP v5 to EZSP v6 was done in EmberZNet 6.0.0 SDK. The change from EZSP v6 to EZSP v7 was in EmberZNet 6.4.0 SDK. EmberZNet 6.7.0 SDK added EZSP v8 (and Secure EZSP Protocol Version 2).
```

Comment:
Again, suggest to remove this as it's not relevant to users of the system.

It feels like you've copied this text from somewhere else?